README.md

```diff
@@ -96,3 +96,5 @@ SimpleCov.start
 ```
```

Comment:
<img width="615" alt="Screenshot 2022-02-07 at 10 44 26" src="https://user-images.githubusercontent.com/95872492/152773473-56de7e5e-21d9-4e2b-bba4-343632663f84.png">
It successfully takes me to the next page and brings my submitted name. Good stuff!